libfreerdp/core/update.c

```diff
@@ -2041,6 +2041,13 @@ static void update_free_queued_message(void* obj)
 	update_message_queue_free_message(msg);
 }
 
+static void update_free_window_state(WINDOW_STATE_ORDER window_state)
+{
+	free(window_state.titleInfo.string);
```

Comment:
Would suggest makeing ```window_state``` a pointer and set it to ```NULL``` after free. That way an invalid access will crash the application reliably.